Makefile

```diff
@@ -53,6 +53,8 @@ check:
 
 test:
 	./test/runner
+	# Check dependencies are consistent for truffleruby and graalvm
+	diff share/ruby-install/graalvm/dependencies.txt share/ruby-install/truffleruby/dependencies.txt
```

Comment:
I think we should add another task called `lint` to do various sanity/QA checks.

Reply:
Sounds good, will do.

Reply:
Done

Reply:
@eregon random question, could the dependencies of truffleruby vs truffleruby-graalvm ever differ? If they absolutely must stay locked, I am considering moving the diff into it's own shunit2 test. Also, we could symlink `truffleruby-graalvm/dependencies.txt` back to `truffleruby/dependencies.txt` to ensure they are always the same.

Reply:
I guess they might differ at some point, if it became necessary.
This check is intended to remind us to check dependencies of one when changing the other.